snuba/cli/migrations.py

```diff
@@ -130,3 +134,68 @@ def reverse(
         raise click.ClickException(str(e))
 
     click.echo(f"Finished reversing migration {migration_key}")
+
+
+@migrations.command()
+@click.option(
+    "--type", "node_type", type=click.Choice(["local", "dist"]), required=True
+)
+@click.option(
+    "--storage-set",
+    "storage_set_names",
+    type=click.Choice([s.value for s in StorageSetKey]),
+    required=True,
+    multiple=True,
+)
+@click.option("--host-name", type=str, required=True)
+@click.option("--port", type=int, required=True)
+@click.option("--user", type=str, required=True, default="default")
+@click.option("--password", type=str, required=True, default="")
+@click.option("--database", type=str, required=True, default="default")
```

Comment:
Shouldn't these be readable from the config ? Also it is not great to have a password taken from the command line as it would be visible in clear in system tools like `ps`

Reply:
I changed this to look for environment variables as well (the same ones used in the default cluster config). We can't exactly take it from the cluster like elsewhere because by definition the node isn't in the cluster yet. I still kept the ability to pass values on the command line because I think it may be useful. Let me know if you would prefer this option be completely removed.